coldfront/core/allocation/templates/allocation/allocation_change.html

```diff
@@ -0,0 +1,137 @@
+{% extends "common/base.html" %}
+{% load crispy_forms_tags %}
+{% load static %}
+
+
+{% block title %}
+Request Allocation Change
+{% endblock %}
+
+
+{% block content %}
+
+<h2>Request change to {{ allocation.get_parent_resource }} for project: {{ allocation.project.title }}</h2>
+<hr>
+
+<p class="text-justify">
+  Request changes to an existing allocation using the form below. For each change
+  you must provide a justification.
+</p>
+
+  <div class="card mb-3">
+    <div class="card-header">
+        <h3><i class="fas fa-list" aria-hidden="true"></i> Allocation Information</h3>   
+    </div>
+
+    <div class="card-body">
+      <form action="{% url 'allocation-change' allocation.pk %}" method="post">

```

Comment:
The HTML is not valid after rendering this view. Looks like we have to move the `<form>` tag up a bit. Here's a patch that will fix the imbalanced tags:

```diff
--- a/coldfront/core/allocation/templates/allocation/allocation_change.html
+++ b/coldfront/core/allocation/templates/allocation/allocation_change.html
@@ -18,13 +18,13 @@ Request Allocation Change
   you must provide a justification.
 </p>
 
+<form action="{% url 'allocation-change' allocation.pk %}" method="post">
   <div class="card mb-3">
     <div class="card-header">
         <h3><i class="fas fa-list" aria-hidden="true"></i> Allocation Information</h3>   
     </div>
 
     <div class="card-body">
-      <form action="{% url 'allocation-change' allocation.pk %}" method="post">
         {% csrf_token %}
         <div class="table-responsive">
           <table class="table table-bordered table-sm">

```